packages/fiori-annotation-api/test/unit/cds/writer.test.ts

```diff
@@ -772,6 +772,115 @@ annotate S.E with {
     ];`
                 );
             });
+            test('by default inserting at the end when deleting property', async () => {
+                const fixture = `
+    Service S { entity E {}; };
+    annotate S.E with @UI.LineItem : [
+        {
+            $Type : 'UI.DataField',
+            Value : test,
+            Criticality : #Critical
+        },
+    ];`;
+                await testWriter(
+                    fixture,
+                    [
+                        {
```

Comment:
I would encourage using factory functions for creating changes. It makes tests a bit more concise and avoids having to repeat the type constants.

Reply:
Changed.